### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

### DISCUSSION THREAD

**Discussion by nimishmedatwal** [community/community#19158] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/91622060/175004445-f34f56a0-dc29-4c6b-8d2f-28feab1dfad3.png)
What can i do to resolbe this issue with github copilot ?

**Comment by rauzerfabiano** [comment]
Hi @nimishmedatwal try this:

1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor

If that doesn't work, so just contacting [Github Support](https://support.github.com/) here.
They would usually respond within a few working days.

Let me know if this helped you.

**Reply by manuarora700** [reply]
Doesnt work

**Reply by rauzerfabiano** [reply]
So just contacting [Github Support](https://support.github.com/) here.

They would usually respond within a few working days.

**Comment by a-eid** [comment]
same issue for me

**Reply by rauzerfabiano** [reply]
Hi @a-eid try this:

1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor

If that doesn't work, so just contacting [Github Support](https://support.github.com/) here.
They would usually respond within a few working days.

Let me know if this helped you.
